spec/index.html

```diff
@@ -1155,6 +1543,245 @@ <h3>Examples</h3>
+          <object data="shared-hashes.svg" style="width: 75%" type="image/svg+xml" aria-describedby="fig-nd-shared-hashes-alt">
+            <p id="fig-nd-shared-hashes-alt">
+              The image represents the graph described in
+              <a href="#ex-nd-shared-hashes-input">
+                the following code block
+              </a>.</p>
+          </object>
+          <figcaption>An illustration of a graph resulting in shared hashes.<br/>
+            Image available in
+            <a href="shared-hashes.svg" title="SVG image of a graph resulting in shared hashes">
+              <abbr title="Scalable Vector Graphics">SVG</abbr>
+            </a>.</figcaption>
+        </figure>
+          
+        <pre id="ex-nd-shared-hashes-input"
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in shared hashes">
+          <!--
+            :p :q ****_:b0**** .
+            :p :q ****_:b1**** .
+            ****_:b0**** :p _:b2 .
+            ****_:b1**** :p _:b3 .
+            _:b2 :r _:b3 .
+          -->
+        </pre>
+
+        <p>The algorithm will be called twice, with each blank node
+           which did not result in a unique hash
+          from the <a href="#hash-1d-quads-algorithm">Hash First Degree Quads Algorithm</a>
+          (`_:b0` and `_:b1`).
+          The <a data-cite="INFRA#map-entry">map entry</a> for `_:b0` in the <a>blank node to quads map</a> results in the
+          following quads:</p>
+
+        <pre id="ex-nd-shared-hashes-b0-quads" data-transform="updateExample">
+          <!--
+            :p :q _:b0 .
+            _:b0 :p _:b2 .
+          -->
+        </pre>
+
+        <p><a href="#hndq-3">Step 3</a> iterates over each of these quads
+          to find <a>blank node</a> <var>components</var>
+          to populate <var>H<sub>n</sub>
+          with the result of running the
+          <a href="#hash-related-blank-node">Hash Related Blank Node algorithm</a>
+          using the <var>position</var> of that <var>component</var> within the quad.</p>
+
+        <table id="ex-nd-shared-table-related-hash-b0"
+               class="hash-related numbered">
+          <caption>Hash for `_:b2`  related to `_:b0`</caption>
+          <thead><tr>
+            <th><var>related</var></th>
+            <th><var>quad</var></th>
+            <th><var>position</var></th>
+            <th><var>hash</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td>`_:b2`</td>
+              <td>`_:b0 :p _:b2 .`</td>
+              <td>`o`</td>
+              <td>
+                <!-- canonical identifer _:b2 is _:c14n0 -->
+                <!-- hash "o<http://example.com/#p>_c14n0"-->
+                <code>bf6431b1112bf1874e44107bfa9169d1213c0a5c35daca0bc62ab72d96214e9e</code></td>
+            </tr>
+          </tbody>
+        </table>
+
```

Comment:
I am not sure how to interpret the table. I would expect to see `o<:p>:b2` which is what gets hashed. I am not sure what I see here.

Reply:
As stated earlier on, the examples use a TriG expression, where the implied BASE and PREFIX are established earlier. So, to get thee hash, expand `:p` to `http://example.com/#p`. The alternative would be using fully expanded IRIs in examples, which IMO would make them less easy to understand.

Reply:
No, the problem is not the prefix form, I completely agree that it should be used. But I think I know why I was messed up. Here is how the table appears on my screen (Safari on MacOS, but it is similar for Firefox):

![Screen Shot 2022-11-30 at 08 43 41](https://user-images.githubusercontent.com/520723/204737073-e2bb60b5-dfb0-4f0f-ac79-f4f81687c460.png)

There are two issues:

- The labels are not separated by column, so it reads "related quad" and "position hash", which of course is rubbish. - Also, the quad column gets a new line, ie, instead of

```
_:b0 :p _:b2 .
```

it is

```
_:b0 :p 
_:b2 .
```

which, at first glance, looks again rubbish.

I am not sure how to really solve this; I am painfully aware that the new W3C style imposes those narrow widths on the main text ☹️

Now that I realized what is really there, the table makes of course sense!

Reply:
One solution might be to use an `abbr` tag for hashes so they display like `29cf..60fa`, which would likely render the table with better column widths.

Reply:
Table styles can be set with padding and similar and/or the column headings could be set to align-center which would have sufficiently similar separation effect, here ... and if the new W3C stylesheet doesn't offer or include such, maybe we can get away with adding it to the CSS for our doc(s) — and/or convince the folks in charge of the new W3C stylesheet to add it there.